src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -68,7 +68,16 @@
       (compojure/GET "/services" []
         (let [statuses (call-status-fns status-fns-atom)]
           {:status 200
-           :body {"services" statuses}})))))
+           :body statuses}))
+       (compojure/GET "/services/:service-name" [service-name]
+         (if-let [status-info (get (deref status-fns-atom) service-name)]
+           (let [status (call-latest-status-fn-for-service status-info)]
+             {:status 200
+              :body (assoc status :service-name service-name)})
+           {:status 404
+            :body {:error {:type ::service-not-found
```

Comment:
I'm not sure if I like namespaces ending up in the response.  Perhaps just `:service-not-found` here instead?